src/roadmap/admin/notificator.py

```
@@ -0,0 +1,85 @@
+
+
+@router.post("/notificator/custom", summary="Trigger lifecycle notification for one or more orgs")
+async def trigger_notificator_custom(payload: CustomNotificatorRequest = Body(...)):
+    org_ids = [payload.org_ids] if isinstance(payload.org_ids, int) else payload.org_ids
+    unique_org_ids = sorted(set(org_ids))
+    await _trigger_lifecycle_notification(org_ids=unique_org_ids)
+    return {"message": "Lifecycle notification completed", "requested_org_ids": unique_org_ids}
+
+
+@router.post("/notificator/all", summary="Trigger lifecycle notification for all orgs", status_code=202)
+async def trigger_notificator_all(background_tasks: BackgroundTasks, payload: AllNotificatorRequest = Body(...)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)